### PR TITLE
fix: Remove encoding character issue beside "PowerShell"

### PR DESCRIPTION
**Pull request by nschonni**
Not sure what character it might have been, but it likly came from a UTF-8 conversion at some point.

Related to https://github.com/MicrosoftDocs/feedback/issues/1574

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR contributed by the community. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR contributed by the community. Thanks.

**Comment by get-itips** [comment]
Yes, it seems an encoding issue. It looks ok

**Comment by tseward** [comment]
Looks good.

File: docset/windows/adrms/adrms.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ ms.assetid: 1CC7729F-B905-4461-A895-3E6A612953AA
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
```
Comment (JohanFreelancer9) [inline docset/windows/adrms/adrms.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```

File: docset/winserver2012-ps/adrms/ADRMS.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ author: kenwith
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/adrms/ADRMS.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```

File: docset/winserver2012r2-ps/adrms/ADRMS.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ ms.assetid: 9451ee96-965a-4221-9403-9c2e760912a2
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/adrms/ADRMS.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```

**Comment by nschonni** [comment]
@JohanFreelancer9 I've added your suggestions

File: docset/winserver2012-ps/adrms/ADRMS.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ author: kenwith
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```
Comment (nenonix) [inline docset/winserver2012-ps/adrms/ADRMS.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets allow you to install, upgrade, or remove a cluster using Windows PowerShell.
```

File: docset/winserver2012-ps/grouppolicy/GroupPolicy.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ author: kenwith
 
 # GroupPolicy Module
 ## Description
-This topic contains the brief descriptions of the Windows PowerShell� cmdlets that are for use in administering Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)
+This topic contains the brief descriptions of the Windows PowerShell cmdlets that are for use in administering Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)
```
Comment (nenonix) [inline docset/winserver2012-ps/grouppolicy/GroupPolicy.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets that used to administer Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets).
```

File: docset/winserver2012r2-ps/adrms/ADRMS.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ ms.assetid: 9451ee96-965a-4221-9403-9c2e760912a2
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```
Comment (nenonix) [inline docset/winserver2012r2-ps/adrms/ADRMS.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets used to deploy in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets allow you to install, upgrade, or remove a cluster using Windows PowerShell.
```

File: docset/winserver2012r2-ps/grouppolicy/GroupPolicy.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ ms.assetid: b9732fda-e344-4d14-8946-791fab93fbaf
 
 # GroupPolicy Module
 ## Description
-This topic contains the brief descriptions of the Windows PowerShell� cmdlets that are for use in administering Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)
+This topic contains the brief descriptions of the Windows PowerShell cmdlets that are for use in administering Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)
```
Comment (nenonix) [inline docset/winserver2012r2-ps/grouppolicy/GroupPolicy.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets used to administer Group Policy in Windows Server and Windows client with Remote Server Administration Tools (RSAT) installed. (RSAT includes the GPMC and the Group Policy cmdlets.)
```

File: docset/windows/adrms/adrms.md [hunk 8]
```diff
@@ -22,7 +22,7 @@ ms.assetid: 1CC7729F-B905-4461-A895-3E6A612953AA
 
 # ADRMS Module
 ## Description
-This topic contains brief descriptions of the Windows PowerShell� cmdlets that are for use in deploying in Windows Server� 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove an cluster using Windows PowerShell.
+This topic contains brief descriptions of the Windows PowerShell cmdlets that are for use in deploying in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets give you the ability to install, upgrade, or remove a cluster using Windows PowerShell.
```
Comment (nenonix) [inline docset/windows/adrms/adrms.md]:
```suggestion
This topic contains brief descriptions of the Windows PowerShell cmdlets used to deploy in Windows Server 2012. Each cmdlet in the table is linked to additional information about that cmdlet. These cmdlets allow you to install, upgrade, or remove a cluster using Windows PowerShell.
```

File: docset/winserver2012r2-ps/wdac/Set-OdbcDriver.md [hunk 9]
```diff
@@ -44,7 +44,7 @@ Specify properties to add or modify by using the **SetPropertyValue** parameter.
 Specify properties to remove by using the **RemovePropertyValue** parameter.
 
 Use the driver installation program to install and uninstall a driver.
-You cannot install or uninstall a driver by using Windows PowerShell� cmdlets.
+You cannot install or uninstall a driver by using Windows PowerShell cmdlets.
```
Comment (nenonix) [inline docset/winserver2012r2-ps/wdac/Set-OdbcDriver.md]:
```suggestion
You cannot install or uninstall a driver using Windows PowerShell cmdlets.
```

File: docset/winserver2012r2-ps/wdac/Set-OdbcDriver.md [hunk 10]
```diff
@@ -44,7 +44,7 @@ Specify properties to add or modify by using the **SetPropertyValue** parameter.
...
 
 For more information about ODBC and drivers, see [Microsoft Open Database Connectivity (ODBC)](http://msdn.microsoft.com/en-us/library/ms710252.aspx) and [Drivers](http://msdn.microsoft.com/en-us/library/ms715383.aspx) on the Microsoft Developer Network.
```
Comment (nenonix) [inline docset/winserver2012r2-ps/wdac/Set-OdbcDriver.md]:
```suggestion
For more information about ODBC and drivers, see [Microsoft Open Database Connectivity (ODBC)](http://msdn.microsoft.com/library/ms710252.aspx) and [Drivers](http://msdn.microsoft.com/library/ms715383.aspx) on the Microsoft Developer Network.
```

**Comment by dariomws306** [comment]
Looks fine!

**Comment by e0i** [comment]
@AndreaBarr The community contributed PR that has passed review can be closed. Thanks.

**Comment by e0i** [comment]
Closes #1100

**Pull request merged**